client/src/components/ui/MetricsSelect.svelte

```diff
@@ -1,12 +1,23 @@
 <script lang="ts">
     import { MetricsMode } from '../types.ts';
+    import { checkPerms } from './forms/permissions/util.ts';
+    import { Global, Local } from '../../../../model/src/permission.ts';
+    import { User } from '~model/user.ts';
+    import { Staff } from '~model/staff.ts';
+
     export let value: MetricsMode | undefined;
+    export let localPermission: Staff['permission'] | null;
+    export let globalPermission: User['permission'] | null;
 </script>
 
 <select required bind:value>
     <option value={MetricsMode.User}>User Summary</option>
-    <option value={MetricsMode.Local}>Local Summary</option>
-    <option value={MetricsMode.Global}>Global Summary</option>
+    {#if !localPermission || checkPerms(localPermission, Local.ViewMetrics)}

```

Comment:
This can be removed by making `localPermission` a required attribute.
```suggestion
    {#if checkPerms(localPermission, Local.ViewMetrics)}
```